CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ The changes are relative to the previous release, unless the baseline is specifi
   AVIF specification.
 * Add an optional argument to the --depth flag of avifenc used to enable a bit
   depth extension scheme in the encoded file.
+* Add a "quality to quantizer (QP)" mapping formula designed for AOM_TUNE_IQ.
```

Comment:
Optional: This probably should go into the "Changed since 1.3.0" section instead because this isn't added to the public interface.